### PR TITLE
Ronny/nlp

### PR DESCRIPTION
**Pull request by rok-tel**

### CODE REVIEW

File: arm-templates/azuredeploy-ctm.json [hunk 1]
```diff
@@ -75,601 +75,609 @@
...
     },
     "isSecondary": {
-      "type":"bool",
+      "type": "bool",
       "defaultValue": false
     }
   },
-  "variables": {    
+  "variables": {
     // Duplicated resources
```
Comment (shahary) [inline arm-templates/azuredeploy-ctm.json:83]: this is illegal in json, i believe

Comment (shahary) [reply arm-templates/azuredeploy-ctm.json:83]: same for all similar comments

File: arm-templates/azuredeploy-fhir.json [hunk 2]
```diff
@@ -193,12 +194,15 @@
             "tags": "[parameters('resourceTags')]",
             "name": "[variables('serviceName')]",
             "location": "[resourceGroup().location]",
+            "dependsOn": [
+              "[variables('appServicePlanResourceId')]"
```
Comment (yochail) [inline arm-templates/azuredeploy-fhir.json:198]: how can the service plan be dependent upon itself?

File: arm-templates/azuredeploy-logicapps.parameters.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
         "isLiveId": {
             "value": false
         },
-        "functionalTestsFileUrl": {
+		    "functionalTestsFileUrl": {
```
Comment (yochail) [inline arm-templates/azuredeploy-logicapps.parameters.json]: alignment

**Review by arieschwartzman — APPROVED** [review]

File: arm-templates/azuredeploy-ctm.parameters.json [hunk 4]
```diff
@@ -32,7 +32,7 @@
         "disqualifierImageTag": {
             "value": "2020.0528.1"
         },
-        "conParserImageTag": {
+        "nlpServicesImageTag": {
```
Comment (yochail) [inline arm-templates/azuredeploy-ctm.parameters.json:35]: did you change the image name itself in different PR?

File: arm-templates/azuredeploy-ctm.json (hunk 1 above)
Comment (yochail) [reply arm-templates/azuredeploy-ctm.json:83]: its true but arm template is letting this slide

File: arm-templates/azuredeploy-ctm.parameters.json (hunk 4 above)
Comment (rok-tel) [reply arm-templates/azuredeploy-ctm.parameters.json:35]: yes

**Pull request merged**